tests/queries/0_stateless/02532_send_logs_level_test.sh

```diff
@@ -8,7 +8,7 @@ CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
 # shellcheck source=../shell_config.sh
 . "$CURDIR"/../shell_config.sh
 
-$CLICKHOUSE_CLIENT -nm -q "
+$CLICKHOUSE_CLIENT -m -q "
```

Comment:
From the looks of it `-m` is also not needed

Reply:
True but let's keep this PR simple and stupid. We can do other cleanups separately.